lib/gitinfo/metrics.go

```diff
@@ -1,19 +1,22 @@
 package gitinfo
 
-import "github.com/prometheus/client_golang/prometheus"
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
 
 var (
-	commitGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	commitGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: "lib",
 		Subsystem: "git",
 		Name:      "commit",
-		Help:      "Constant gauge with label 'commit' set to the current git commit hash.",
+		Help:      "Constant gauge with label 'commit' set to the build info git commit hash.",
 	}, []string{"commit"})
 
-	timestampGauge = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	timestampGauge = promauto.NewGauge(prometheus.GaugeOpts{
 		Namespace: "lib",
 		Subsystem: "git",
-		Name:      "timestamp",
-		Help:      "Constant gauge with label 'timestamp' set to the current git commit timestamp.",
-	}, []string{"timestamp"})
+		Name:      "timestamp_unix",
+		Help:      "Build info git commit timestamp in unix seconds.",
```

Comment:
We need only one gauge - `up` and it should have all the other data as labels: `git_commit`, `go_version`, and other important versions, e.g. cosmos SDK version etc.

Reply:
`up` is built in, can't change it

Reply:
ok, then we use the build in, but we still need only one gauge, e.g. `build_info` and add all other labels to that one gauge. If we use the built-in it should have the go version out of the box.